### PR TITLE
Enforce valid ComputeTaskDef

### PR DESCRIPTION
**Pull request by jiivan**
fixes: #36

### CODE REVIEW

File: tests/test_basic.py [hunk 1]
```diff
@@ -169,3 +170,18 @@ def test_invalid_task_to_compute(self):
             s = msg.serialize()
             msg2 = message.Message.deserialize(s, decrypt_func=None)
             self.assertIs(msg2, None)
+
+
+class ComputeTaskDefTestCase(unittest.TestCase):
+    def test_type(self):
+        ctd = message.ComputeTaskDef()
+        ctd['src_code'] = ("Nie mamy w Polsce monarchy,"
```
Comment (maaktweluit) [inline tests/test_basic.py]: I cant review this part, should it be in polish?

File: golem_messages/message.py [hunk 2]
```diff
@@ -697,6 +697,12 @@ def __init__(self, compute_task_def=None, **kwargs):
         self.compute_task_def = compute_task_def
         super().__init__(**kwargs)
 
+    def deserialize_slot(self, key, value):
+        value = super().deserialize_slot(key, value)
+        if key == 'compute_task_def':
+            value = ComputeTaskDef(value)
```
Comment (maaktweluit) [inline golem_messages/message.py]: what happens when there is an error? is this caught and logged on a higher level?

Comment (jiivan) [reply golem_messages/message.py]: Yes, in `.deserialize()` https://github.com/golemfactory/golem-messages/blob/master/golem_messages/message.py#L252

File: tests/test_basic.py (hunk 1 above)
Comment (jiivan) [reply tests/test_basic.py]: :) It's just a poem by Tadeusz Boy-Żeleński. If you feel uncomfortable with this I can remove it. It does not affect the test.

Comment (maaktweluit) [reply tests/test_basic.py]: Cool! no need, maybe a comment above it that it is just test data / lorem ipum would be enough :)

**Comment by mfranciszkiewicz** [comment]
Please create a release ASAP, currently it is not possible to compute tasks in develop.

**Pull request merged**